test/Microsoft.Azure.WebJobs.Host.FunctionalTests/TableTests.cs

```diff
@@ -143,7 +143,7 @@ public void Table_IfBoundToICollectorJObject_AddInsertsEntity()
             Assert.NotNull(entity.Properties);
 
             AssertPropertyValue(entity, "ValueStr", "abcdef");
-            AssertPropertyValue(entity, "ValueNum", 123);         
```

Comment:
I use https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.ProductivityPowerTools2015. There's a `Format document on save` and `Remove and Sort Usings on save` that fixes spacing issues when I save a file.